### DISCUSSION TITLE
希望下个版本可以增加批量复制链接的功能

### DISCUSSION THREAD

**Discussion by louiejordan** [lsky-org/lsky-pro#619] [想法]
不是刚上传后批量复制，而是在已上传的大量图片里批量复制链接，我认为这是个很基础的功能，奈何兰空居然没有。

**Comment by heiyuan0801** [comment]
可以一键选择的

**Reply by louiejordan** [reply]
免费版的没有啊，只有付费版的可以，我就自己一个人用，没必要买付费的啊，这种很基础的功能居然还要花钱我是挺意外的。

**Reply by 0xxb** [reply]
> 免费版的没有啊，只有付费版的可以，我就自己一个人用，没必要买付费的啊，这种很基础的功能居然还要花钱我是挺意外的。

付费版也是后来更新慢慢加上去的，并非为了付费不给开源版本适配，个人精力有限，望理解。

**Reply by louiejordan** [reply]
免费版也加上吧，有这个功能就完整了。我现在都不敢存太多图片，不然后期整理很麻烦。
